scripts/nlp_language_modeling/build_knn_map_index.py

```diff
@@ -175,6 +264,8 @@ def get_emb():
         default='bert-base-nli-mean-tokens',
```

Comment:
@yidong72 I thought you were now using `all-mpnet-base-v2`

Reply:
yes. this is just the default argument. When using this script, I set it to `all-mpnet-base-v2`

Reply:
sounds good